ciscripts/site-gem-vulnerability-test/cibuild

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -e # halt script on error
+
```

Comment:
Here, and in the other script, make this independent of the directory it is run from:

```bash
readonly HERE="$(cd $(dirname $0) && pwd)"

cd ${HERE}/../../site # probably need to change this after you move the script
```